Bindings/Python/tests/test_DataTable.py

```diff
@@ -88,6 +88,9 @@ def test_DataTable(self):
                 row2[2] == row[2] and
                 row2[3] == row[3])
         print table
+        # Reorder columns of the table.
+        table.reorderColumns([3, 1, 0, 2])
+        table.reorderColumns([2, 1, 3, 0])
```

Comment:
Not a rigorous test, as it will still pass if `table.reorderColumns()` does nothing.